src/plugins/intel_cpu/src/emitters/plugin/x64/jit_load_store_emitters.cpp

```diff
@@ -676,7 +677,8 @@ void jit_store_emitter::emit_data() const {
 }
 
 void jit_store_emitter::emit_impl(const std::vector<size_t> &in_idxs, const std::vector<size_t> &out_idxs) const {
-    const int offset = in_idxs.size() == 2 ? in_idxs[1] : 0;
+    // offset in store emitter is the offset of dst gpr register, should be parsed from out_ids.
```

Comment:
Minor: The best place for this comment in the corresponding header file before class definitions 😃 Another developer will take a look at interface and didn't find the description for `offset`. And only after searching in files, he will find this comment.